packages/repository/test/unit/decorator/model-and-relation.ts

```diff
@@ -102,11 +102,10 @@ describe('model decorator', () => {
 
   it('adds property metadata', () => {
     const meta = Reflector.getOwnMetadata(
```

Comment:
Can we use `MetadataInspector` in these tests please? That way the tests are decoupled from the internal implementation details of `*DecoratorFactory`.